src/goto-harness/recursive_initialization.cpp

```diff
@@ -74,6 +74,17 @@ bool recursive_initialization_configt::handle_option(
       COMMON_HARNESS_GENERATOR_MIN_ARRAY_SIZE_OPT, values);
     return true;
   }
+  else if(option == COMMON_HARNESS_GENERATOR_FUNCTION_POINTER_CAN_BE_NULL_OPT)
+  {
+    std::transform(
+      values.begin(),
+      values.end(),
+      std::inserter(
```

Comment:
For the Windows build, you are going to have to `#include <iterator>`

Reply:
Thanks, I figured it'd be something like that!